src/components/Button/Button.scss

```
@@ -698,9 +698,9 @@ $stacking-order: (
     }
 
     &.newDesignLanguage {
-      @include focus-ring($border-width: border-width('base'));
       border-color: currentColor;
       box-shadow: 0 0 0 border-width('base') currentColor;
+      @include focus-ring($border-width: rem(2px));
```

We could use the thick variant of `border-width` here as well.

```suggestion
      @include focus-ring($border-width: border-width('thick'));
```

---

This is a good idea, I noticed we don't do it in lots of places. I think a separate PR to clean that up would be useful.